templates/index.html

```diff
@@ -198,7 +198,7 @@ <h3>Subscribe to Vanilla updates</h3>
         <div class="p-form__group">
```

Comment:
As @lyubomir-popov mentioned. The headings are inconsistent here, so the `<h3>Subscribe to Vanilla updates</h3>` should be changed to `h2` as well.